core/compatible.py

```diff
@@ -122,9 +97,8 @@ def check_for_requirements(start_api_server):
     Returns:
         True if exist otherwise False
     """
-    from core.alert import messages
-    from config import api_configuration
     # check external required modules
+    connection_timeout = config.api_configuration()["api_database_connection_timeout"]
```

Comment:
Since the returned value of config.api_configuration() doesn't change, I feel that we can reduce the number of function calls to this function by storing the result in a variable and using that instead.